DuckDuckGo/DuckPlayer/DuckPlayerNavigationHandler.swift

```diff
@@ -108,6 +108,9 @@ final class DuckPlayerNavigationHandler {
             return
         }
         
+        // This setting allows the FE to skip showing the overlay on some videos
```

Comment:
Certainly! Here's the improved version of the comment:

"I think this comment could be improved. 'Some videos' is a bit unclear. Does the FE apply any heuristic to decide when to show or not show the overlay? Or is it simpler, as in, if we send this as true, the FE doesn't display the overlay?

The property name itself could also be improved. 'allowFirstVideo' is not very clear to me. What is a 'first video'? If the flag allows the native side to hide the overlay, maybe something like 'skipOverlay' would be better?"